toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1613,7 +1614,25 @@ func ProvisionUserSSHCerts(installChroot safechroot.ChrootInterface, username st
 	}
 	defer os.Remove(authorizedKeysTempFile)
 
-	allSSHKeys := make([]string, 0, len(sshPubKeyPaths)+len(sshPubKeys))
+	allSSHKeys := []string(nil)
+
+	if includeExistingKeys {
+		authorizedKeysFileFullPath := filepath.Join(installChroot.RootDir(), authorizedKeysFile)
+
+		fileExists, err := file.PathExists(authorizedKeysFileFullPath)
```

Comment:
Just to be even more specific and avoid cases when someone does something weird like make this file a directory:
```suggestion
		fileExists, err := file.IsFile(authorizedKeysFileFullPath)
```

Reply:
`IsFile` returns an error if the file doesn't exist, which isn't what I want.

Reply:
Ah, right, sorry.